src/test/lifecycle/common.js

```diff
@@ -63,8 +64,31 @@ function date_lifecycle_configuration(Bucket, Key) {
 }
 exports.date_lifecycle_configuration = date_lifecycle_configuration;
 
+function past_date_lifecycle_configuration(Bucket, Key) {
```

Comment:
Could you reuse the previous definition of `date_lifecycle_configuration` and make the change you need on this function's output?

Reply:
Done. Thanks